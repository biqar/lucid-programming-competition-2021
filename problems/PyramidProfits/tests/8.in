50
f4131db57e d88bd294bf 70
d88bd294bf 7afa61321c 85
5fd89ff7fa a2a8f8425c 85
9e4e8cd1b6 565bf1b8fa 75
15103569b1 108269a3f4 25
3f5a927cad 34294c3aac 60
3180b8d981 c1f626fbdb 35
6f08716148 629124caab 50
acfbd09396 a2a8f8425c 70
629124caab b099e1e41c 55
565bf1b8fa ce786b3c8f 60
c0f8a7389a b099e1e41c 75
eac4822bd7 Jude 65
136f19f002 5190e7d99f 80
ca18e60176 eb216c26f9 25
a2a8f8425c c1f626fbdb 30
7afa61321c ff74354e9f 85
59dbe30af3 acfbd09396 90
eb216c26f9 a2a8f8425c 85
5051eb29bb d55ade53b5 70
ce786b3c8f Jude 80
3e4b084af1 Jude 20
6a04ea59cf 9e4e8cd1b6 85
d55ade53b5 15103569b1 5
b83b416580 ff74354e9f 10
c540519e0c 8326fff28f 35
77b02c14f1 ce786b3c8f 10
34294c3aac 5190e7d99f 20
ff74354e9f Jude 60
1a737f1193 5fd89ff7fa 30
763d771d6e ce786b3c8f 30
b099e1e41c af39cac6f6 35
108269a3f4 Jude 40
041729084a c1f626fbdb 20
dbdf4a81a7 d55ade53b5 60
50c98a691c acfbd09396 65
c1f626fbdb ce786b3c8f 25
2a4bcd2d37 aead9a8598 20
c50c914363 ca18e60176 30
f8bf60e89c 59dbe30af3 45
a209156060 2587212fa8 70
2587212fa8 c0f8a7389a 45
4ed1910ee5 5fd89ff7fa 30
be75be3d72 9e4e8cd1b6 65
5190e7d99f 2a4bcd2d37 10
5a444e92ec eac4822bd7 25
a3dff5ce02 a2a8f8425c 10
aead9a8598 5051eb29bb 85
8326fff28f aead9a8598 50
af39cac6f6 Jude 75
500
aead9a8598 862000
ff74354e9f 1042600
5fd89ff7fa 1706500
a3dff5ce02 1829900
108269a3f4 1207000
dbdf4a81a7 1777500
136f19f002 520900
acfbd09396 1449700
a2a8f8425c 1104500
c50c914363 931000
59dbe30af3 1994800
eb216c26f9 65500
50c98a691c 561500
c50c914363 1658400
ff74354e9f 232200
565bf1b8fa 1206500
d88bd294bf 1284500
d55ade53b5 1461800
b099e1e41c 756000
565bf1b8fa 677600
1a737f1193 1182600
565bf1b8fa 45700
5190e7d99f 1757000
aead9a8598 1131100
b099e1e41c 814100
5fd89ff7fa 1405300
ce786b3c8f 1668000
eb216c26f9 693600
041729084a 1136100
c50c914363 1447600
b099e1e41c 1316700
dbdf4a81a7 602800
a209156060 1296200
ca18e60176 390800
5190e7d99f 99400
5051eb29bb 1159600
ca18e60176 1486500
5a444e92ec 783100
2587212fa8 1197300
a3dff5ce02 132700
d55ade53b5 569800
59dbe30af3 845600
9e4e8cd1b6 200600
9e4e8cd1b6 18900
3180b8d981 1620400
acfbd09396 1352300
b83b416580 983900
aead9a8598 1079500
a209156060 1956000
a2a8f8425c 507400
c540519e0c 657700
5190e7d99f 1886300
3e4b084af1 1126200
3e4b084af1 1539600
9e4e8cd1b6 719200
041729084a 1148300
a209156060 396000
77b02c14f1 1505900
3f5a927cad 13300
3f5a927cad 1564500
77b02c14f1 287000
ca18e60176 276800
f8bf60e89c 1198800
7afa61321c 403100
041729084a 1129700
f4131db57e 1063600
d55ade53b5 1280200
Jude 1381000
2587212fa8 1997800
dbdf4a81a7 1283700
acfbd09396 126500
Jude 245400
a3dff5ce02 1288500
f8bf60e89c 1715700
136f19f002 523900
5fd89ff7fa 1205900
6a04ea59cf 1950000
c540519e0c 1812700
34294c3aac 1342300
5fd89ff7fa 1836000
5a444e92ec 219400
6f08716148 1832000
a2a8f8425c 1747100
c540519e0c 1422700
5190e7d99f 1174100
59dbe30af3 289000
4ed1910ee5 1857900
b099e1e41c 1698300
5a444e92ec 1821300
a209156060 140000
d55ade53b5 634800
d88bd294bf 957900
9e4e8cd1b6 125800
f4131db57e 250700
7afa61321c 1717400
Jude 1746200
5fd89ff7fa 885300
a3dff5ce02 80300
6f08716148 1909400
dbdf4a81a7 1096900
629124caab 1970500
1a737f1193 145600
c0f8a7389a 1051400
eb216c26f9 660100
ca18e60176 1466600
dbdf4a81a7 1689500
59dbe30af3 1456200
108269a3f4 1514800
c50c914363 909000
Jude 1170200
4ed1910ee5 1762100
eb216c26f9 365700
565bf1b8fa 313500
77b02c14f1 748100
a209156060 1721100
ff74354e9f 1694900
c0f8a7389a 758600
f8bf60e89c 385100
2587212fa8 1198100
c0f8a7389a 286100
af39cac6f6 1294800
eb216c26f9 270700
eb216c26f9 885200
f4131db57e 783400
9e4e8cd1b6 448900
dbdf4a81a7 1679800
ff74354e9f 1748500
be75be3d72 279500
7afa61321c 350100
5a444e92ec 737200
77b02c14f1 1297100
ce786b3c8f 1798700
3180b8d981 1502500
8326fff28f 1352800
Jude 1916700
f8bf60e89c 1048500
Jude 107700
f8bf60e89c 1613200
aead9a8598 439700
77b02c14f1 1492000
eb216c26f9 1118000
f4131db57e 680700
763d771d6e 521700
1a737f1193 494900
c50c914363 1427100
7afa61321c 287800
041729084a 587200
f8bf60e89c 1907600
108269a3f4 1509800
8326fff28f 1738100
c0f8a7389a 1613000
c0f8a7389a 893800
c0f8a7389a 1536400
3f5a927cad 624200
3e4b084af1 899600
763d771d6e 739300
2a4bcd2d37 1008900
a2a8f8425c 1178700
ce786b3c8f 1468200
3f5a927cad 1845600
a3dff5ce02 1161900
108269a3f4 790900
77b02c14f1 509000
9e4e8cd1b6 1741400
ca18e60176 612500
f4131db57e 723500
59dbe30af3 778100
c1f626fbdb 1780900
dbdf4a81a7 1670600
aead9a8598 455000
041729084a 1154000
acfbd09396 100800
dbdf4a81a7 1666100
629124caab 1859000
3f5a927cad 1446700
041729084a 121400
a3dff5ce02 91500
6f08716148 1706200
eac4822bd7 1948800
50c98a691c 896600
2a4bcd2d37 721900
aead9a8598 134200
77b02c14f1 827200
f4131db57e 342100
c1f626fbdb 49800
c1f626fbdb 623100
be75be3d72 672400
3f5a927cad 1705000
a2a8f8425c 595900
3180b8d981 202700
c0f8a7389a 62900
15103569b1 684100
136f19f002 1195400
a209156060 626100
7afa61321c 1774300
34294c3aac 438800
d88bd294bf 456000
2a4bcd2d37 1389100
2587212fa8 751800
ca18e60176 1332300
5190e7d99f 1256000
629124caab 239600
ce786b3c8f 153900
5a444e92ec 481500
4ed1910ee5 279200
5a444e92ec 71000
eac4822bd7 1084800
136f19f002 852400
d55ade53b5 38700
c50c914363 1736700
15103569b1 534000
136f19f002 1349400
d88bd294bf 385300
5fd89ff7fa 1389300
50c98a691c 1410000
9e4e8cd1b6 1987400
34294c3aac 277800
041729084a 603300
acfbd09396 941000
041729084a 64900
77b02c14f1 1996400
77b02c14f1 903100
4ed1910ee5 366700
c540519e0c 1581200
3f5a927cad 1288000
f4131db57e 1601900
c50c914363 1525200
041729084a 1956400
a2a8f8425c 353300
59dbe30af3 713600
50c98a691c 698200
b83b416580 602100
15103569b1 1074200
2587212fa8 278500
3e4b084af1 1288400
eb216c26f9 1568200
acfbd09396 1690900
Jude 646200
3180b8d981 994200
7afa61321c 924200
aead9a8598 1049900
ff74354e9f 761300
3f5a927cad 549800
2587212fa8 12200
5051eb29bb 1418500
50c98a691c 1844800
b83b416580 765100
565bf1b8fa 1738800
34294c3aac 1164900
1a737f1193 1927200
a3dff5ce02 1626100
6f08716148 1104200
Jude 83800
2587212fa8 1141200
4ed1910ee5 249000
6f08716148 325000
c1f626fbdb 899900
5fd89ff7fa 758100
3e4b084af1 111800
be75be3d72 1274400
d88bd294bf 591600
aead9a8598 1760600
8326fff28f 1974200
d88bd294bf 952300
ff74354e9f 1561300
1a737f1193 765800
77b02c14f1 1999400
c540519e0c 1897000
Jude 596500
77b02c14f1 1464600
b099e1e41c 1166300
a2a8f8425c 550900
2587212fa8 1517200
b83b416580 577400
af39cac6f6 753300
be75be3d72 319800
d55ade53b5 1964100
eac4822bd7 874900
34294c3aac 956800
3e4b084af1 297000
eac4822bd7 251000
b099e1e41c 139900
8326fff28f 1106100
1a737f1193 1853900
ce786b3c8f 1675900
2a4bcd2d37 845500
8326fff28f 1383900
a2a8f8425c 1616900
acfbd09396 296500
5051eb29bb 1793400
5190e7d99f 1287800
4ed1910ee5 1533900
59dbe30af3 1740400
7afa61321c 545100
af39cac6f6 1895500
763d771d6e 1303700
108269a3f4 1656200
d55ade53b5 958600
763d771d6e 1410100
Jude 231100
ca18e60176 1447200
565bf1b8fa 1159700
ff74354e9f 1479000
136f19f002 897100
9e4e8cd1b6 1816900
59dbe30af3 251600
d88bd294bf 183700
5051eb29bb 1667100
5a444e92ec 1326400
565bf1b8fa 151700
dbdf4a81a7 722000
af39cac6f6 543100
5a444e92ec 324400
a209156060 1502000
ff74354e9f 1723800
Jude 1502600
108269a3f4 864500
59dbe30af3 706100
5fd89ff7fa 902600
5051eb29bb 592200
eac4822bd7 1855100
c540519e0c 1362200
34294c3aac 1545100
5fd89ff7fa 1049200
a3dff5ce02 1283300
3180b8d981 982100
59dbe30af3 497900
b099e1e41c 1982100
6a04ea59cf 1918300
2587212fa8 860300
b099e1e41c 42100
ff74354e9f 521500
acfbd09396 1240500
f8bf60e89c 1738000
565bf1b8fa 990300
9e4e8cd1b6 810400
acfbd09396 1887200
acfbd09396 683100
3180b8d981 733400
eb216c26f9 302300
5190e7d99f 556100
6a04ea59cf 632200
3e4b084af1 798100
d55ade53b5 1961300
5a444e92ec 998600
8326fff28f 461700
a209156060 1541300
7afa61321c 449600
5fd89ff7fa 635200
f8bf60e89c 1873600
59dbe30af3 851300
136f19f002 871200
6f08716148 652000
a209156060 805200
d55ade53b5 637200
d88bd294bf 682100
d55ade53b5 500400
108269a3f4 1203700
c1f626fbdb 1597000
af39cac6f6 618200
f8bf60e89c 1970000
50c98a691c 1319000
59dbe30af3 793700
c0f8a7389a 1591800
ff74354e9f 1420700
1a737f1193 896100
15103569b1 834200
eac4822bd7 283300
ca18e60176 452400
7afa61321c 941300
763d771d6e 888800
5fd89ff7fa 1770600
c50c914363 686200
5051eb29bb 1737400
eb216c26f9 1539200
108269a3f4 3600
c1f626fbdb 222100
136f19f002 1796600
7afa61321c 657700
7afa61321c 930800
9e4e8cd1b6 1435200
77b02c14f1 375200
77b02c14f1 91200
5051eb29bb 1757700
a2a8f8425c 1024800
eac4822bd7 860800
Jude 1831100
c1f626fbdb 1649600
f4131db57e 686300
c50c914363 371800
Jude 1349700
8326fff28f 316200
ce786b3c8f 647400
f8bf60e89c 370300
3e4b084af1 197600
34294c3aac 544300
3f5a927cad 1953700
c540519e0c 899400
be75be3d72 86500
a3dff5ce02 1057600
d55ade53b5 1392500
f8bf60e89c 1937000
629124caab 898500
a209156060 1537800
5a444e92ec 693400
ff74354e9f 183300
dbdf4a81a7 1983000
59dbe30af3 1021100
eb216c26f9 250900
c0f8a7389a 693300
15103569b1 959900
34294c3aac 1755400
3e4b084af1 1201500
629124caab 960100
3f5a927cad 76100
c50c914363 742800
ca18e60176 413100
Jude 1266900
6a04ea59cf 133600
c540519e0c 1212600
eb216c26f9 902100
9e4e8cd1b6 1110000
3f5a927cad 1642400
d88bd294bf 1921100
c540519e0c 181400
6a04ea59cf 1109100
6f08716148 615200
f8bf60e89c 182700
b83b416580 1798200
5a444e92ec 653600
108269a3f4 1731500
5a444e92ec 1682400
5a444e92ec 1409100
b83b416580 1478600
136f19f002 1292200
15103569b1 334100
108269a3f4 881600
59dbe30af3 756300
5190e7d99f 542000
3e4b084af1 1429700
dbdf4a81a7 813300
f8bf60e89c 1355400
50c98a691c 428800
f4131db57e 1787500
dbdf4a81a7 812500
629124caab 1082500
5051eb29bb 1606900
f8bf60e89c 1985300
108269a3f4 1936400
c540519e0c 1700500
5fd89ff7fa 490500
eac4822bd7 1684000
7afa61321c 1153700
aead9a8598 1741600
9e4e8cd1b6 241000
b83b416580 1947700
c1f626fbdb 268600
6f08716148 1232400
5190e7d99f 1115900
6f08716148 1660100
c50c914363 168100
f8bf60e89c 1168800
7afa61321c 448000
565bf1b8fa 894300
f4131db57e 1072500
b099e1e41c 713800
c540519e0c 889200
eb216c26f9 133700
6a04ea59cf 1304500
565bf1b8fa 350600
15103569b1 142600
2a4bcd2d37 1055400
5051eb29bb 1220400
77b02c14f1 1351400
af39cac6f6 1927800
77b02c14f1 1816200
af39cac6f6 864200
9e4e8cd1b6 1288000
8326fff28f 780500
6a04ea59cf 812700
eac4822bd7 483000
3f5a927cad 31500
7afa61321c 451100
041729084a 543600
5a444e92ec 1056900
77b02c14f1 1947700
eac4822bd7 90200
be75be3d72 1908700
50c98a691c 1742000
1a737f1193 1166800
34294c3aac 479300
f8bf60e89c 1403200
15103569b1 690200
a209156060 589000
b83b416580 931500
a209156060 1610800
1a737f1193 255900
629124caab 1058500
5a444e92ec 850000
3180b8d981 179600
