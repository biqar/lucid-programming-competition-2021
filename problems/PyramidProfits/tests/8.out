041729084a 337796
108269a3f4 41928190
136f19f002 443
15103569b1 11618860
1a737f1193 132939
2587212fa8 1157996
2a4bcd2d37 24525
3180b8d981 435043
34294c3aac 162
3e4b084af1 1777900
3f5a927cad 91
4ed1910ee5 92547
5051eb29bb 85564
50c98a691c 242995
5190e7d99f 1698
565bf1b8fa 6535224
59dbe30af3 934225
5a444e92ec 2130099
5fd89ff7fa 1241892
629124caab 1961616
6a04ea59cf 2405282
6f08716148 796697
763d771d6e 1167264
77b02c14f1 1488920
7afa61321c 6882555
8326fff28f 26722
9e4e8cd1b6 5296397
Jude 188917220
a209156060 850947
a2a8f8425c 2453581
a3dff5ce02 51311
acfbd09396 686226
aead9a8598 167227
af39cac6f6 34635304
b099e1e41c 6567078
b83b416580 545070
be75be3d72 1062664
c0f8a7389a 4126312
c1f626fbdb 21421581
c50c914363 44385
c540519e0c 7089
ca18e60176 204012
ce786b3c8f 121779168
d55ade53b5 222958
d88bd294bf 4383621
dbdf4a81a7 47393
eac4822bd7 12510631
eb216c26f9 1191706
f4131db57e 2728683
f8bf60e89c 371723
ff74354e9f 14796771
